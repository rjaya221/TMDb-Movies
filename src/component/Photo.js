import React from 'react';

 var Photo=(data)=>{
            return(<div><img className="single-photo" src={"https://image.tmdb.org/t/p/w500/"+data.url}  alt="No Display"/>
            </div>);
}
export default Photo;