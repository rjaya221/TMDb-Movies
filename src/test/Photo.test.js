import React from 'react';
import { shallow } from '../enzyme';
import Photo from '../component/Photo';

describe("should rendered Photo", () => {
  it("should render my component", () => {
    
   // const wrapper = shallow(<Photo url="/dzBtMocZuJbjLOXvrl4zGYigDzh.jpg"/>);
   // expect(wrapper.getElements()).toMatchSnapshot();
  });
});
