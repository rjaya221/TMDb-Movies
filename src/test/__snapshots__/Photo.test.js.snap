// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rendered Photo should render my component 1`] = `
Array [
  <div>
    <img
      alt="No Display"
      className="single-photo"
      src="https://image.tmdb.org/t/p/w500//dzBtMocZuJbjLOXvrl4zGYigDzh.jpg"
    />
  </div>,
]
`;
